.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.formatting.provider": "black"
+}
```

Comment:
Is there a specific reason why we want to have that top level now instead of inside the api folder?

Reply:
I moved it manually, but is the one in the subfolder really unused? VsCode has a concept called folder settings so you can override settings for each folder manually and independent of the user settings. So I thought it would fit putting this setting in the api folder, but Im also fine with keeping it at root ^^

Reply:
Yeah, on my machine it finds the one in the submodule and then copies it into the superfolder^^

I don't really care where we put it either 🤷‍♂️

Reply:
Lets leave it how it is now then :D